server/src/logic.js

```diff
@@ -291,6 +291,20 @@ export const groupHandler = {
       }),
     );
   },
+  removeUserFromGroup(_, args, ctx) {
+    const { groupId } = args.groupUpdate;
+    return getAuthenticatedUser(ctx).then(user =>
+      Group.findById(groupId).then((group) => {
```

Comment:
we'll obviously need to do some permissions checks here but dw about it right now

Reply:
Please try yank out e5f092d :-)